releasetasks/test/test_releasetasks.py

```diff
@@ -521,3 +503,46 @@ def test_final_verify_task_definition(self):
             "queue:task-priority:high",
         ])
         self.assertTrue(expected_graph_scopes.issubset(graph["scopes"]))
+
+    def test_multi_channel_final_verify_task_definition(self):
+        graph = make_task_graph(
+            version="42.0b2",
+            appVersion="42.0",
+            buildNumber=3,
+            source_enabled=False,
+            en_US_config={"platforms": {
+                "linux": {"task_id": "xyz"},
+                "win32": {"task_id": "xyy"}
+            }},
+            l10n_config={},
+            repo_path="releases/foo",
+            revision="fedcba654321",
+            branch="foo",
```

Comment:
wondering if we even need to pass branch to final verify template now? maybe instead just change this to channel: https://github.com/mozilla/releasetasks/blob/master/releasetasks/templates/final_verify.yml.tmpl#L26 ?